dotCMS/src/main/java/com/dotcms/util/I18NUtil.java

```diff
@@ -81,15 +81,15 @@ private I18NUtil() {}
         final Map<String, String> messagesMap = map();
 
         if (null != messagesKey) {
-
+            final Language lang = APILocator.getLanguageAPI().getLanguage(locale.getLanguage(), locale.getCountry());
             messagesKey.forEach(
                     messageKey -> {
 
                         try {
-
+                            
                             messagesMap.put(messageKey,
-                                    LanguageUtil.get(locale, messageKey));
-                        } catch (LanguageException e) {
+                                    APILocator.getLanguageAPI().getStringKey(lang, messageKey));
+                        } catch (Exception e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)